docs/synctl-update-test.md

```diff
@@ -101,6 +112,40 @@ synctl update test <synthetic-id> \
     --expect-status 200
 ```
 
+Expect json example
+```
+synctl update <synthetic-id> \
+    --expect-json '{
+      "slideshow": {
+        "author": "Yours Truly",
+        "date": "date of publication",
+        "slides": [
+          {
+            "title": "Wake up to WonderWidgets!",
+            "type": "all"
+          },
+          {
+            "items": [
+              "Why <em>WonderWidgets</em> are great",
+              "Who <em>buys</em> WonderWidgets"
+            ],
+            "title": "Overview",
+            "type": "all"
+          }
+        ],
+        "title": "Sample Slide Show"
+      }
+    }'
+```
+Update test with expect-match, expect-exists and expect-not-empty
+
+```
+synctl update test <synthetic-id> \
+    --expect-match ibm \
+    --expect-exists '["slideshow"]' \
+    --expect-not-empty '["slideshow"]'
+```
```

Comment:
It is not recommended users to use these three options in a single test, although it can do. Let's seperate it to three examples with different options.